scripts/admin/apache-proxy/ssl.conf

```diff
@@ -0,0 +1,50 @@
+<IfModule mod_ssl.c>
+	SSLPassPhraseDialog  exec:/usr/share/apache2/ask-for-passphrase
+	SSLSessionCache		shmcb:${APACHE_RUN_DIR}/ssl_scache(512000)
+	SSLSessionCacheTimeout  300
+    SSLRandomSeed startup file:/dev/urandom  256
+    SSLRandomSeed connect builtin
+    SSLCryptoDevice builtin
+
+    <VirtualHost _default_:443>
+        ErrorLog logs/ssl_error_log
+        TransferLog logs/ssl_access_log
+        LogLevel warn
+        SSLEngine on
+        # Require TLS 1.2 only
+        SSLProtocol -all +TLSv1.2
+        # Require Strong Ciphers
+        SSLCipherSuite "EECDH:!RC4:!3des:!SHA"
+        # Require Cipher Order
+        SSLHonorCipherOrder on
+        SSLCertificateFile <path to cert file provided by CA>
+        SSLCertificateKeyFile <path to key file>
```

Comment:
It's really hard to find the path in this file.  Please add 

# CHANGE PATH HERE

Or something that makes sense to help user find this.

Reply:
That was not supposed to be bold 

```
# CHANGE PATH HERE
```

Reply:
Added some additional comments to make these options stand out more